litedram/core/multiplexer.py

```diff
@@ -136,11 +136,11 @@ def __init__(self, txxd):
         # # #
 
         if txxd is not None:
-            count = Signal(max=max(txxd+1, 2))
+            count = Signal(max=max(txxd, 2))
```

Comment:
If we want to be able to load txxd value in the signal, len must be txxd +1

Reply:
You are correct, but then the fix stops working so I think the bug is more complex.  Getting closer to a solution.

Reply:
This is now correct because we set the count to txxd-1